testool/src/statetest/yaml.rs

```diff
@@ -778,6 +778,7 @@ arith:
         Ok(())
     }
 
+    #[cfg(feature = "warn-unimplemented")]
```

Comment:
:suspect:

Reply:
I'm not sure what that means LOL

@adria0 told me to put this as the test should have never had this one enabled